frontend/ui/widget/bookmapwidget.lua

```diff
@@ -632,13 +632,17 @@ function BookMapWidget:init()
     self.covers_fullscreen = true -- hint for UIManager:_repaint()
 
     if Device:hasKeys() then
-        self.key_events = {
-            Close = { { Input.group.Back } },
-            ScrollRowUp = { { "Up" } },
-            ScrollRowDown = { { "Down" } },
```

Comment:
You're removing scrolling with Up/Down on the emulator there - and adding the need to use Shift with Up/Down below.
Oversight or on purpose?

Reply:
fixed...?

Reply:
Seems so.